### ISSUE TITLE
Custom Bot name as an argument for inline comment editing

### ISSUE THREAD

**Issue by danielloader** [sbdchd/squawk-action#40]
We have this action in our PR checks as one expects, though developers including myself often forget to put a branch into draft PR mode and thus keep pushing commits onto an open PR - even when we don't, rebasing off the upstream is a regular task which triggers the PR check runs.

Would it be possible to update a single comment in the PR comment chain in place rather than adding a new comment each time the run is triggered? 

Other jobs do this in our PR review process so felt it was worth asking so it's less verbose in our PR comment threads.

**Comment by danielloader** [comment]
Just read the CLI source a bit and found something I assume is the cause for it not working for me - after I found it should be...

```rust
  match comments.iter().find(|x| {
      x.user.r#type == "Bot"
          && x.user.login == bot_name            // "github-actions[bot]" - not what our bot is called
          && x.body.contains(existing_comment_text_includes) 
  }) {
      Some(prev_comment) => gh.update_issue_comment(...),  
      None              => gh.create_issue_comment(...),   
  }
```

I'm using the Github actions application login method, but via the official `uses: actions/create-github-app-token@v1` step first to make a token (because multiple of our steps emit comments instead of using the native mechanism provided by the ENV vars for squawk).

I think I've sort of fixed it myself by taking the outputs of the step and wiring them into the envs for the squawk step but ideally I'd be able to hand a scoped token that's short lived to the action and set the bot name rather than handing over the full app id and private key into a third party action.

```yaml
        - name: Squawk
          if: ${{ steps.migrations.outputs.files != '' }}
          uses: sbdchd/squawk-action@8d113898e16c6cfd417ceba848b0edad97882be3 # v2.1.0
          env:
            SQUAWK_GITHUB_APP_ID: ${{ vars.REPO_SERVICES_ACCESS_APP_ID }}
            SQUAWK_GITHUB_INSTALL_ID: ${{ steps.app-token.outputs.installation-id }}
            SQUAWK_GITHUB_PRIVATE_KEY: ${{ secrets.REPO_SERVICES_ACCESS_PRIVATE_KEY }}
          with:
            files: ${{ steps.migrations.outputs.files }}
            version: 2.60.0
            upload-to-github: true
            fail-on-violations: false
```

So technically not blocked I don't think, just would be nice to have the bot name configurable if we want to pass in a pre minted shared app token into multiple comment producing jobs.
